src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1280,12 +1300,16 @@ public function getPricesAction(int $orderId): JsonResponse
     {
         /** @var OrderForViewing $orderForViewing */
         $orderForViewing = $this->getQueryBus()->handle(new GetOrderForViewing($orderId));
+        $orderForViewingPrices = $orderForViewing->getPrices();
+        $zeroNumber = new Number('0');
```

Comment:
used just once. Maybe you dont need to put it in a variable

Reply:
Fixed